curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd1ca72b3e615a33c05b6.md

```diff
@@ -0,0 +1,46 @@
+---
+id: 656cd1ca72b3e615a33c05b6
+title: Task 123
+challengeType: 22
+dashedName: task-123
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Sophie: Well, it is not far.
+-->
+
+# --description--
+
+In English, you can use a certain structure to indicate that something isn't in a particular state or condition in the present.
```

Comment:
In the description, I'd suggest replacing "isn't" with "is not" as in the audio and putting it in backticks to clarify this is what we are trying to explain here.